hypershift-operator/metrics.go

```diff
@@ -330,8 +331,10 @@ func clusterCreationTime(hc *hyperv1.HostedCluster) *float64 {
 	if completionTime == nil {
 		return nil
 	}
-	creationTime := completionTime.Sub(hc.CreationTimestamp.Time).Seconds()
-	return &creationTime
+	if !hc.CreationTimestamp.IsZero() {
```

Comment:
I don't know of a case where the CreationTimestamp can be nil?

Reply:
It seems highly unlikely, but I was just looking for similar cases in the file like `hc.DeletionTimestamp`. I didn't figure it would hurt to have the checks, but I can take them out too if wanted.

Reply:
Unless you can prove me wrong let's drop it, A resource by design can't exist without a CreationTimestamp, I'd expect api machinery to set this before persisting it.

Reply:
Ok the checks on `CreationTimestamp` have been removed.